sound/soc/intel/boards/bxt_tdf8532.c

```diff
@@ -10,6 +10,19 @@
 #include <sound/pcm.h>
 #include <sound/soc.h>
 #include <sound/pcm_params.h>
+#include <sound/jack.h>
```

Comment:
If the DSP is not enabled, we should select another machine driver.

Reply:
that's true, will change like that.